server/src/main/java/org/elasticsearch/rest/Access.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.rest;
+
+public enum Access {
+    PUBLIC,
+    INTERNAL,
+    HIDDEN
```

Comment:
Without wanting to bikeshed a Proof of Concept, I think we had some ambiguity in meaning for `HIDDEN` - some people thought it was equivalent to "internal" (that is, hidden but still exists) but here it means "unavailable" (hidden and cannot be reached).

Could we use `UNAVAILABLE` / `DISABLED` / `BLOCKED` instead?

Reply:
Do we need a name for it here at all? I think we agreed the default state should be blocking handlers, and that simply omitting the annotation would suffice for that? This way we also don't need any gradle magic to ensure all handlers have the annotation.

Reply:
Yeah getting rid of it makes sense (it's not used in the logic at all). I had thought of it as an explicit marker that someone had already looked at this rest handler and decided that it ought to be hidden (vs one that someone just hadn't looked at yet).

Reply:
I have a general preference for explict, but I really don't want two ways to say the same thing (no annotation, or an annotation with "UNAVAILABLE"), so I'm good with dropping it from the enum.